pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -58,6 +60,12 @@ const (
 
 	InfraNodeBootstrapServiceAccountName = "node-bootstrapper"
 	NodeBootstrapRoleName                = "system:node-bootstrapper"
+
+	InfraTemplateInstanceControllerServiceAccountName = "template-instance-controller"
+	TemplateInstanceControllerRoleName                = "system:openshift:controller:template-instance-controller"
+
+	InfraTemplateServiceBrokerServiceAccountName = "template-service-broker"
```

Comment:
let's put comments on each of these SA constants explaining the responsibilities/role of the SA.  Will help people trying to evaluate the permissions being granted in the future.